__tests__/index.tests.js

```diff
@@ -39,3 +43,155 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => { return fs.readFile('./endpoints.json', "utf-8").then((endpoints)=>{
+        const endpoints1=JSON.parse(endpoints)
+        
+         
+          return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            
+            expect(response.body.endpoints).toEqual(expect.any(Object));
```

Comment:
Agree with August's comment, the test on line 56 to expect any object isn't a good test. Maybe remove this and keep the object.keys test